Tests/test_file_apng.py

```diff
@@ -689,3 +689,13 @@ def test_different_modes_in_later_frames(mode, default_image, duplicate, tmp_pat
     )
     with Image.open(test_file) as reloaded:
         assert reloaded.mode == mode
+
+
+def test_apng_issue_7700():
+    # https://github.com/python-pillow/Pillow/issues/7700
+    with Image.open("Tests/images/apng/issue_7700.png") as im:
```

Comment:
Please could you give the test and image more descriptive names than `issue_7700`?